app/herdbook.py

```diff
@@ -293,74 +347,73 @@ def get_ind_inbreeding(i_number, g_id):
+                                    settings.rapi.port,
+                                    g_id),
+                            params={})
 
     if response.status_code == 200:
         return csvparser.parse_csv(response.content)
 
-    else:
-        APP.logger.error("Could not fetch inbreeding data.")
-        APP.logger.error("Error {}".format(response))
-        return {}
+    APP.logger.error("Could not fetch inbreeding data.")
+    APP.logger.error("Error {}".format(response))
+    return {}
 
 
 @APP.route("/api/<int:g_id>/kinship/")
 def kinship(g_id):
-    id = str(g_id)
-    kinship = get_kinship(id)
-    return jsonify(kinship)
+    return jsonify(get_kinship(str(g_id)))
 
 
 def get_kinship(g_id):
-    response = requests.get('http://{}:{}/kinship/{}'.format(settings.rapi.host, settings.rapi.port, g_id),
-                             params={})
+    response = requests.get('http://{}:{}/kinship/{}' \
+                            .format(settings.rapi.host,
+                                    settings.rapi.port,
+                                    g_id),
+                            params={})
 
     if response.status_code == 200:
         return csvparser.parse_kinship(response.content)
 
-    else:
-        APP.logger.error("Could not fetch kinship data.")
-        APP.logger.error("Error {}".format(response))
-        return {}
+    APP.logger.error("Could not fetch kinship data.")
+    APP.logger.error("Error {}".format(response))
+    return {}
 
 
 def get_ind_mean_kinship(i_number, g_id):
     """
     Returns the mean kinship coefficient of the individual given by `i_number`.
     In case the individual is not active, we return 0.
     """
-    mean_kinship = get_mean_kinship(g_id)
-    if i_number in mean_kinship:
-        return mean_kinship[i_number]
-    return 0
+    mk_values = get_mean_kinship(g_id)
+    return mk_values[i_number] if i_number in mk_values else 0
 
 
 @APP.route("/api/<int:g_id>/kinship/")
 def mean_kinship(g_id):
-    id = str(g_id)
-    mean_kinship = get_mean_kinship(id)
-    return jsonify(mean_kinship)
+    return jsonify(get_mean_kinship(str(g_id)))
 
 
 @APP.route("/api/<int:g_id>/meankinship/")
 def get_mean_kinship(g_id):
-    response = requests.get('http://{}:{}/meankinship/{}'.format(settings.rapi.host, settings.rapi.port, g_id),
-                             params={})
+    response = requests.get('http://{}:{}/meankinship/{}' \
+                            .format(settings.rapi.host,
+                                    settings.rapi.port,
+                                    g_id),
+                            params={})
 
     if response.status_code == 200:
         return csvparser.parse_csv(response.content)
 
-    else:
-        APP.logger.error("Could not fetch mean kinship data.")
-        APP.logger.error("Error {}".format(response))
-        return {}
+    APP.logger.error("Could not fetch mean kinship data.")
+    APP.logger.error("Error {}".format(response))
+    return {}
```

Comment:
Flag error clearer?

Reply:
How do you mean? Also - I didn't write this function, I just cleaned it up, so I don't think I'm the right person to clarify things ^^

Reply:
It might be for the error issue rather, but it feels like this should flag an error clearer to the caller (but then again, maybe an empty message is that error).